src/token/CustomFixedFee.js

```diff
@@ -38,6 +39,35 @@ export default class CustomFixedFee extends CustomFee {
         }
     }
 
+    /**
+     * @param {Hbar} amount
+     * @returns {CustomFixedFee}
+     */
+    setHbarAmount(amount) {
+        this._amount = amount.toTinybars();
+        this._denominatingTokenId = null;
+        return this;
+    }
+
+    /**
+     * @returns {Hbar} 
+     */
+    get hbarAmount() {
```

Comment:
Can we update this implementation to return `null` if `this._denominatingTokenId` is not `null`? Otherwise if a user does `CustomFixedFee.setDenominatingTokenId("0.0.10").setAmount(10).hbarAmount` they'd get a non-null value implying the `CustomFixedFee` is a Hbar amount.
Note: Your implementation is in line with Java's, and I'll make an issue to update Java as well.